pkg/controllers/counter/controller.go

```diff
@@ -74,16 +74,20 @@ func (c *Controller) resourceCountsFor(ctx context.Context, provisionerName stri
 	if err := c.kubeClient.List(ctx, &nodes, client.MatchingLabels{v1alpha5.ProvisionerNameLabelKey: provisionerName}); err != nil {
 		return nil, err
 	}
-	var cpu = resource.NewScaledQuantity(0, 0)
-	var memory = resource.NewScaledQuantity(0, resource.Giga)
+
```

Comment:
I'm wondering if we can collapse this logic into the core provisioning loop. Trying to remember why it was decoupled.